pkgs/by-name/si/simde/package.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "simde";
+  version = "0.7.6";
+
+  src = fetchFromGitHub {
+    owner = "simd-everywhere";
+    repo = "simde";
+    rev = "v${version}";
+    hash = "sha256-pj+zaD5o9XYkTavezcQFzM6ao0IdQP1zjP9L4vcCyEY=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/include
+    cp -a ${pname} $out/include
+
+    install -Dt $out/share/doc/${pname} README.md
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    homepage = "https://simd-everywhere.github.io";
+    description = "Implementations of SIMD instruction sets for systems which don't natively support them.";
+    license = with licenses; [mit];
+    platforms = platforms.unix;
+  };
```

Comment:
Would you mind adding yourself as package maintainer? I'm not able to find you in the maintainer list, so maybe you'll have to add yourself there as well?